test/e2e/tests/tolerate-failure/login/asset-details-power-user.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { generateWalletState } from '../../../../../app/scripts/fixtures/generate-wallet-state';
+import { WITH_STATE_POWER_USER } from '../../../benchmarks/constants';
+import { withFixtures } from '../../../helpers';
+import { loginWithoutBalanceValidation } from '../../../page-objects/flows/login.flow';
+import AssetListPage from '../../../page-objects/pages/home/asset-list';
+import HomePage from '../../../page-objects/pages/home/homepage';
+import NetworkManager from '../../../page-objects/pages/network-manager';
+import { Driver } from '../../../webdriver/driver';
+import { setupTimerReporting } from '../utils/testSetup.js';
+import Timers from '../../../../timers/Timers.js';
```

Comment:
```suggestion
import Timers from '../../../../timers/Timers.ts';
```

Reply:
hi there 👋 @javiergarciavera this issue is still open. I think the we should commit Danica's suggestion. Wdyt? 🤔